ihmc-pub-sub/src/main/java/us/ihmc/pubsub/publisher/Publisher.java

```diff
@@ -23,9 +23,12 @@
 /**
  * Class Publisher, used to send data to associated subscribers.
  *
+ * Deprecated - use {@link us.ihmc.pubsub.impl.fastRTPS.FastRTPSPublisher} directly
+ *
  * @author Jesper Smith
  *
  */
+@Deprecated
 public interface Publisher
```

Comment:
Do we plan on removing this in the future?

Reply:
Yes